README.md

```diff
@@ -125,28 +125,52 @@ For more examples, see the [examples](examples) directory. For more information
 
 ### Create a model
 
-`ollama create` is used to create a model from a Modelfile.
+Create a model from a Modelfile
+
+```
+ollama create <modelname> -f ./Modelfile
+```
 
 ### Pull a model
 
+Pull a model from the library at [ollama.ai/library](https://ollama.ai/library).
+
 ```
 ollama pull llama2
 ```
 
-> This command can also be used to update a local model. Only the diff will be pulled.
-
 ### Remove a model
 
+Remove the model files from your local system.
```

Comment:
I think we can do without this one

Reply:
Remove model all together or just the explanation?

Reply:
Actually I think all the explanations of cli commands can be left out.